specification.md

```diff
@@ -54,6 +57,11 @@ where:
     -   https://theupdateframework.com/Root/v1.0.5
     -   etc...
 
+*   KEYID is an optional, unauthenticated hint indicating what key was used to
```

Comment:
While I realise that signing-spec does not currently define `Sign()` or talk about how to use KEYID...

Is KEYID currently optional in in-toto? Do we anticipate that the KEYID is only omitted in cases where there is only one valid key? Can we provide some additional guidance here?

Reply:
I actually don't know about the in-toto side. Perhaps some in-toto expert can answer? Also, please note that full specification of KEYID and Sign() is covered by issue #9.

Please take a look at the most recent version. I expanded the text to hopefully better explain:
- It is negotiated out of band.
- It is only used to narrow the selection of keys.

Do you think this is sufficient for the time being?

Reply:
I do, this looks good thanks Mark!